src/state/session/index.tsx

```diff
@@ -551,41 +547,59 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
     return persisted.onUpdate(() => {
       const session = persisted.get('session')
 
-      logger.debug(`session: onUpdate`, {}, logger.DebugContext.session)
+      logger.info(`session: persisted onUpdate`, {})
```

Comment:
Sending these to Sentry as breadcrumbs so we know if a broadcast resulted in a logout